package.json

```diff
@@ -11,6 +11,7 @@
     "@giphy/js-fetch-api": "^4.0.1",
     "@sentry/browser": "^7.51.2",
     "@sentry/integrations": "^7.51.2",
+    "@types/gtag.js": "^0.0.12",
```

Comment:
`@types/gtag.js` should be added as a `devDependency`. You can do that by using `pnpm install --save-dev @types/gtag.js` (shorthand `pnpm i -D @types/gtag.js`)

Reply:
Adjusted accordingly. Please check!